include/zephyr/devicetree.h

```diff
@@ -2203,6 +2203,14 @@
  */
 #define DT_REG_ADDR(node_id) DT_REG_ADDR_BY_IDX(node_id, 0)
 
+/**
+ * @brief 64-bit version of a DT_REG_ADDR macro.
+ *        Use it in cases when 64-bit address value may appear in Device Tree
```

Comment:
"devicetree" is preferred to "device tree"; please use this instead.

The `@brief` block should generally be one line; please separate this second line and do not indent it.

Reply:
Got it, thanks!